build.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e -u -o pipefail
 
-readonly MRUSTC_VER='b5b7089'
+readonly MRUSTC_VER='master'
```

Comment:
Don't change this to master. Instead, use the hash of the commit that is currently HEAD of master.